e2e/README.md

```diff
@@ -20,10 +20,10 @@ After that in the other (second) terminal type
 
 to the terminal.
 
-With command above the tests run in headless chrome. To run tests in other browser(s) you should use the BROWSER environment variable, eg.:
+With command above the tests run in headless chrome. To run tests in other browser(s) you should use the E2EPROFILE environment variable, eg.:
 
-- `BROWSER=firefox npm run test:e2e`
-- `BROWSER=firefox,chrome npm run test:e2e`
+- `E2EPROFILE=firefox npm run test:e2e`
```

Comment:
please don't forget to update the `.env.example` as well!

Reply:
Sorry, what to update in env.example?

Reply:
ahhh sorry I thought we documented the `BROWSER` and `TEST_PROVIDER` in the `.env.example`. nevermind.